packages/tupaia-web/src/features/Chart.tsx

```diff
@@ -3,51 +3,128 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import styled from 'styled-components';
-import { Chart as ChartComponent, ViewContent } from '@tupaia/ui-chart-components';
+import { Chart as ChartComponent, ChartTable, ViewContent } from '@tupaia/ui-chart-components';
+import { BarChart, GridOn } from '@material-ui/icons';
+import { Tabs } from '@material-ui/core';
+import { Tab } from '@material-ui/core';
+import { TabContext, TabPanel } from '@material-ui/lab';
 
-const Wrapper = styled.div<{
-  $isEnlarged: boolean;
-  $hasData: boolean;
-}>`
+const Wrapper = styled.div`
   display: flex;
   position: relative;
   align-content: stretch;
   -webkit-box-align: stretch;
   align-items: stretch;
-  height: ${({ $isEnlarged, $hasData }) => {
-    if (!$hasData) return 'auto';
-    return $isEnlarged ? '22.5rem' : '14rem';
-  }};
   flex-direction: column;
   .recharts-responsive-container {
     min-width: 0px;
   }
   .recharts-wrapper {
     font-size: 1rem !important; // this is to make sure the labels on the charts are relative to the base font size
   }
-  // Make the charts conform to the parent container's size
-  .recharts-wrapper,
-  .recharts-wrapper svg {
-    height: 100% !important;
-    width: 100%;
-  }
   li.recharts-legend-item {
     white-space: nowrap; // ensure there are no line breaks on the export legends
   }
 `;
 
+const TabsWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+
+  .MuiTabs-indicator {
+    display: none;
+  }
+`;
+
+const TabsGroup = styled(Tabs)`
+  border: 1px solid
+    ${({ theme }) => {
+      let hexString = theme.palette.text.primary.replace('#', '');
+      hexString = hexString.length === 3 ? hexString.replace(/(.)/g, '$1$1') : hexString;
+      return `#${hexString}33`;
```

Comment:
This seems a bit clunky to me and maybe a bit fragile too as the primary text color could be 3 digit hex, 6 digit hex or also rgba which is what it is actually by default in the [default theme](https://v4.mui.com/customization/default-theme/#default-theme). Maybe we could use lighten instead?